recipe/meta.yaml

```diff
@@ -36,17 +36,18 @@ test:
 
 about:
   home: https://www.freetype.org/
```

Comment:
`home` URL should be without **www** to prevent redirection: `https://freetype.org/`